site/data/authors.json

```diff
@@ -9,4 +9,5 @@
   "jroweboy": { "key": "jroweboy", "name": "jroweboy", "avatar": "https://community.citra-emu.org/user_avatar/community.citra-emu.org/jroweboy/120/30_1.png" },
   "jmc47": { "key": "jmc47", "name": "JMC47", "avatar": "https://community.citra-emu.org/user_avatar/community.citra-emu.org/citrabot/120/28_1.png" },
   "anodium": { "key": "anodium", "name": "anodium", "avatar": "https://community.citra-emu.org/user_avatar/community.citra-emu.org/anodium/120/485_1.png"}
-}
+  "Saphiresurf": {"key": "Saphiresurf", "name": "Saphiresurf", "avatar": "https://community.citra-emu.org/user_avatar/community.citra-emu.org/saphiresurf/120/695_1.png"},
```

Comment:
Syntax error on line 12:

 * missing a comma on line 11
 * extra comma at the end of line 12
 * missing terminating newline at line 13